transcription/transcription.go

```diff
@@ -29,3 +30,15 @@ func msgHeaders(from string, to []string, subject string) string {
 	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
 	return strings.Join(msgHeaders, "\r\n")
 }
+
+// ConvertAudioIntoRequiredFormat converts encoded audio into the required format.
+func ConvertAudioIntoRequiredFormat(fn string) error {
```

Comment:
I am supposed to mock `exec.Command()` and `Cmd.Run()` from `exec`. Not really sure how to run `EXPECT()` with a non-static method